src/main/java/qna/domain/Answer.java

```diff
@@ -11,8 +11,12 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long writerId;
-    private Long questionId;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "writerId")
+    private User writer;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "questionId")
+    private Question question;
```

Comment:
요구사항의 외래키 이름과 다른 부분이 있네요 ! 다른 부분도 확인 부탁드릴게요 :)

추가적으로 개인 취향이지만 엔티티에서 변수 사이에 공백이 있으면 가독성이 조금 더 좋아진다고 생각합니다 😄 
```suggestion
    @JoinColumn(name = "writer_id")
    private User writer;
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "question_id")
    private Question question;
```